falcon/routing/util.py

```diff
@@ -140,6 +140,11 @@ def map_http_methods(resource):
             should define ``on_get(self, req, resp)`` and
             ``on_post(self, req, resp)``.
 
+        suffix: A string used to look for alternate methods to the base HTTP methods.
+            If suffix is specified Falcon will look for resource methods ending in suffix,
+            e.g. if suffix is 'foo', then a "GET" request will be mapped to
```

Comment:
``"GET"`` and ``"POST"`` are quoted as ``GET`` and ``'GET'`` in this same docstring, maybe it would be worth making the consistent?

Reply:
yeah, that makes sense, I made the consistent across the docstring with my latest commit